packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/services/EditingContextCrossReferenceAdapter.java

```diff
@@ -108,7 +108,7 @@ private Collection<Setting> getNonContainmentReferences(Collection<Setting> inve
             EStructuralFeature eStructuralFeature = setting.getEStructuralFeature();
             if (eStructuralFeature instanceof EReference) {
                 EReference eReference = (EReference) eStructuralFeature;
-                if (!eReference.isContainment()) {
+                if (!eReference.isContainment() && !eReference.isDerived()) {
                     containmentReferences.add(setting);
```

Comment:
I had not noticed the first time, but... this method computes and return *non*-containment references, and puts them in a variable named `containmentReferences`.

Reply:
Done